packages/universal-components/src/MenuGroup/MenuItemWrapper.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import Separator from './Separator';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+type Props = {|
+  +children: React.Node,
+  +iconRendered?: boolean,
+|};
+
+export default function MenuItemWrapper({ children, iconRendered }: Props) {
+  return (
+    <View>
+      <View style={styles.wrapper}>{children}</View>
+      <Separator style={iconRendered ? styles.noIcon : styles.iconSeparator} />
```

Comment:
Separator margin still seems to be slightly off on iOS.

<img width="382" alt="Screenshot 2019-03-11 at 14 25 39" src="https://user-images.githubusercontent.com/2660330/54127282-b4c95a80-4409-11e9-82e4-6f692f24ae81.png">

Reply:
Fixed